### DISCUSSION TITLE
Kinda new here but willing to put my efforts to contribute

### DISCUSSION THREAD

**Discussion by manu-r12** [bryceco/GoMap#796] [General]
i wanna know how can i get started ?

**Comment by bryceco** [comment]
Hi Manu! This is a pretty mature project, which means that the "easy" stuff has is mostly done. Also it primarily uses UIKit with just a small amount of SwiftUI. If you want something fairly easy you could try implementing https://github.com/bryceco/GoMap/issues/777. Just be aware that the button can be moved to either side of the screen.
